swagger_py_codegen/base.py

```diff
@@ -55,7 +55,7 @@ def __init__(self):
         self.env = Environment(loader=self.loader)
 
     def add_searchpath(self, path):
-        self.loader.searchpath.append(path)
+        self.loader.searchpath.insert(0, path)
```

Comment:
Will you submit a PR about `template searchpath priority` separately? so we can merge this fleetly.